src/DataStructures/Python/Matrix.cpp

```diff
@@ -0,0 +1,46 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <boost/python.hpp>
+#include <sstream>
+#include <string>
+#include <utility>
+
+#include "DataStructures/Matrix.hpp"
+#include "PythonBindings/BoundChecks.hpp"
+#include <boost/python/reference_existing_object.hpp>
+#include <boost/python/return_value_policy.hpp>
+#include <boost/python/tuple.hpp>
+namespace bp = boost::python;
```

Comment:
please add a blank line before the `namespace bp = ` so it's clearly separate from the includes